sdk/src/lib.rs

```diff
@@ -140,17 +140,18 @@ pub(crate) mod hashed_uri;
 #[allow(dead_code)]
 pub(crate) mod jumbf;
 pub(crate) mod salt;
-pub(crate) mod settings;
+pub mod settings;
 pub(crate) mod status_tracker;
 pub(crate) mod store;
 pub(crate) mod time_stamp;
+pub(crate) mod trust_handler;
 pub(crate) mod utils;
 pub mod validation_status;
 pub use hash_utils::HashRange;
 pub(crate) use utils::{cbor_types, hash_utils};
 pub use utils::{cbor_types::DateT, hash_utils::hash_stream_by_alg};
 pub(crate) mod validator;
-#[cfg(target_arch = "wasm32")]
+//#[cfg(target_arch = "wasm32")]
```

Comment:
why do we want to include wasm always? Can we remove this or add a comment?

Reply:
I don't remember making this change.  Let me look into it.

Reply:
I reverted back